bin/webpack-dev-server

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env ruby
 
 ENV["RAILS_ENV"] ||= ENV["RACK_ENV"] || "development"
-ENV["NODE_ENV"] ||= ENV["RAILS_ENV"]
+ENV["NODE_ENV"]  ||= "development"
```

Comment:
Why change this?

Reply:
This is also a result of the Webpacker upgrade script, but I agree it should be left alone. 👍

Reply:
Same as previous reason. Won't change.